src/test/java/jalau/at18/katas/Team2/Test2048/MainTest.java

```diff
@@ -0,0 +1,15 @@
+package jalau.at18.katas.Team2.Test2048;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MainTest {
+    @Test
+    public void shouldMatrix() {
```

Comment:
If this is empty, better remove this file, it causes noise to have code that is not actually doing something useful.

Reply:
The Main.test file it's already removed :)